iocage_lib/ioc_start.py

```diff
@@ -1181,7 +1181,7 @@ def start_network_vnet_addr(self, iface, ip, defaultgw, ipv6=False):
         if ipv6:
             ifconfig = [iface, 'inet6', ip, 'up']
         else:
-            ifconfig = [iface, ip, 'up']
+            ifconfig = [iface, ip, 'alias']
```

Comment:
You still need the up and the end

Reply:
Nope. Docs say that when ever an ip is added, the interface is automatically changed to `up` state